packages/salesforce/src/Adaptor.js

```diff
@@ -317,7 +317,9 @@ export function bulkQuery(qs, options, callback) {
       qs,
       options
     );
-    const apiVersion = connection.version;
+
+    if (connection.version < '47.0')
```

Comment:
This won't work - you'll need to parse the version into a number

This should do it
```
if (parseFloat(connection.version) < 47)
```

Reply:
Done ✅